hydra/ci-hydra-master.nix

```diff
@@ -0,0 +1,104 @@
+{ config, pkgs, lib, ... }:
+
+let
+  keys = import ./ci-keys.nix;
+in {
+  networking.firewall.allowedTCPPorts =
+    [ config.services.hydra.port ];
```

Comment:
Are you sure this is correct? What does `config.services.hydra.port` evaluate to? As it is, the firewall blocks Internet connections. Changing this to the following opens the service to the Internet.

```
networking.firewall.allowedTCPPorts = [ 22 80 443 ];
```

Reply:
It uses default port of 3000 from hydra config. With your config, updated with your additions.